[ADD] PLM: engineering change order doc

Felicious opened the pull request:
### Summary
Create a new doc that details:
- How to create new ECOs in 3 ways (normal way, tablet view, from an email alias)
- Overview of ECO types and stages
- How to close/apply changes to an ECO

Looking for feedback (in particular) on document organizational structure: given the flow of using ECOs, do the section titles make sense? Are they in an order that makes most sense?

And of course, I welcome feedback on any wording, brevity, accuracy of the flow suggestions :smile:

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/5238).

hojo-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,250 @@
+   :align: center
+   :alt: "Show Instructions" icon in the *Operations* tab of a BoM.
+
+In opened the :ref:`quality control point <quality/quality_control_points>` form, ensure the
```
```suggestion
On the :ref:`quality control point <quality/quality_control_points>` form, ensure the
```

hojo-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,250 @@
+      :align: center
+      :alt: alt text
+
+In the new :abbr:`BoM (Bill of Material)`, switch to the :guilabel:`Operations` tab and select the
```
```suggestion
On the new :abbr:`BoM (Bill of Material)`, switch to the :guilabel:`Operations` tab and select the
```

hojo-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,250 @@
+      :alt: alt text
+
+In the new :abbr:`BoM (Bill of Material)`, switch to the :guilabel:`Operations` tab and select the
+:guilabel:`Show Instructions` icon (:guilabel:`☰ (three horizontal lines)`). Doing so opens a list
```
```suggestion
:guilabel:`☰ (Show Instructions)` icon. Doing so opens a list
```
I like to use the symbol itself followed by the name of the symbol in parentheses (ex. settings).

hojo-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,250 @@
+To ensure the changes are live, navigate to the :guilabel:`Revision` smart button. The :abbr:`BoM
+(Bill of Material)` will have the large red :guilabel:`Archived` tag removed. Additionally, navigate
+to the product form, switch to the :guilabel:`Miscellaneous` tab, and the :guilabel:`Version` field
+will be incremented.
```
It might be good to give a more detailed explanation than just saying "incremented".

hojo-odoo reviewed (commented):
Hey @Felicious,

Good job on this. I've made a few comments that should be applied throughout the doc. Also, one of the .rst files is missing a blank line at the end.

Thanks,
hojo

Felicious commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,250 @@
+.. _plm/eco:
+
+Utilize *engineering change orders* (ECO) to track and implement changes to products and :ref:`bill
+of materials <manufacturing/management/bill-configuration>`. View and create :abbr:`ECOs
```
Oh yes! thanks for catching that

Felicious commented:
Hello @odoo/crm-doc-review (: This Product management lifecycle (PLM) doc is ready for the second round of peer review! If you get to this before 8/10, that's *amazing* because that's when I'll be asking an MRP expert from support to do another review. It'll be so nice if ya'll are able to help me make the doc sound snappier :blush: If not, that's okay! You'll be reviewing an even more accurate and detailed doc :smile:

Thanks in advance for your help!

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,265 @@
+Once the approvers accept the changes, the :guilabel:`Apply Changes` button will become available.
+Click this button, and the |ECO| will be automatically closed and the changes are applied.
+
+To ensure the changes are live, navigate to the :guilabel:`Revision` smart button. The |BOM| will
+have the large red :guilabel:`Archived` tag removed. Additionally, navigate to the product form,
+switch to the :guilabel:`Miscellaneous` tab, and the :guilabel:`Version` field will be updated to
+match the version number shown on the :guilabel:`Revision` smart button of the latest |ECO|.
```
```suggestion
To ensure the changes are live, navigate to the :guilabel:`Revision` smart button.

The |BOM| will have the large red :guilabel:`Archived` tag removed. Additionally, navigate to the
product form, switch to the :guilabel:`Miscellaneous` tab, and the :guilabel:`Version` field will be
updated to match the version number shown on the :guilabel:`Revision` smart button of the latest
|ECO|.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,265 @@
+
+|ECOs| created in *tablet view* are intended for operators to directly suggest clearer operation
+instructions. To create an |ECO|, navigate to the :guilabel:`Work Orders` tab on an ongoing
+manufacturing order. Click the :guilabel:`tablet icon` for the desired work order to open the tablet
+view of the operation.
```
```suggestion
manufacturing order. Then, click the :guilabel:`tablet icon` for the desired work order to open the
tablet view of the operation.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,265 @@
+   :align: center
+   :alt: Find the tablet icon for each operation, second from the far right.
+
+Next, add an instructional step by clicking the :guilabel:`☰ (three horizontal lines)` icon in
+tablet view of an operation. Then, click the :guilabel:`Add a step` button.
```
```suggestion
Next, add an instructional step, by clicking the :guilabel:`☰ (three horizontal lines)` icon in the
tablet view of an operation. Then, click the :guilabel:`Add a step` button.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,265 @@
+
+.. example::
+   To propose an additional check for broken components, enter the details in the :guilabel:`Add a
+   Step` pop-up. Doing so creates an instructional quality control point that will be reviewed in
+   the following section.
```
```suggestion
   Step` pop-up window. Doing so creates an instructional quality control point that will be
   reviewed in the following section.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,265 @@
+--------
+
+To review the proposed changes, go to the `BOM Changes` ECO type from :menuselection:`PLM app -->
+Overview` and click on the card to view the contained |ECOs|. Click on the newly created |ECO| in
+the `New` stage, and view a summary of the proposed changes in the :guilabel:`Operation Changes`
+tab. To implement the changes, click the :guilabel:`Revision` smart button.
```
```suggestion
Overview`, and click on the card to view the contained |ECOs|. Click on the newly created |ECO| in
the `New` stage, and view a summary of the proposed changes in the :guilabel:`Operation Changes`
tab. To implement the changes, click the :guilabel:`Revision` smart button.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,265 @@
+      :align: center
+      :alt: Find the new ECO in the "BOM Changes" ECO type, in the *New* stage.
+
+On the new |BOM|, switch to the :guilabel:`Operations` tab and select the :guilabel:`☰ (Show
+Instructions)` icon. Doing so opens a list of :guilabel:`Steps` to perform the operation, with the
+newest instruction titled `New Step Suggestion:` followed by the user-entered title. Click the line
+item to view the suggested changes.
```
```suggestion
On the new |BOM|, switch to the :guilabel:`Operations` tab, and select the :guilabel:`☰ (Show
Instructions)` icon. Doing so opens a list of :guilabel:`Steps` to perform the operation, with the
newest instruction titled `New Step Suggestion:`, followed by the user-entered title. Click the line
item to view the suggested changes.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,265 @@
+On the :ref:`quality control point <quality/quality_control_points>` form, ensure the following form
+fields are accurately filled out to give detailed instructions for operators:
+
+- :guilabel:`Title`: rename to give a concise description of the new instruction
+- :guilabel:`Control per`: using the drop-down menu, determine whether this instruction applies
+  broadly for the :guilabel:`Product`, specifically for this :guilabel:`Operation` only, or a
+  particular :guilabel:`Quantity` of the product
+- :guilabel:`Type`: categorizes the control point type. From the drop-down menu, select
+  :guilabel:`Instructions` to detail an instruction for the worker. To receive input from the
+  workers, select the :guilabel:`Take a Picture`, :guilabel:`Register Consumed Materials`,
+  :guilabel:`Print Label`, or other :ref:`quality check options <quality/quality_control_points>`.
```
```suggestion
- :guilabel:`Title`: rename to give a concise description of the new instruction.
- :guilabel:`Control per`: using the drop-down menu, determine whether this instruction applies
  broadly for the :guilabel:`Product`, specifically for this :guilabel:`Operation` *only*, or a
  particular :guilabel:`Quantity` of the product.
- :guilabel:`Type`: categorizes the control point type. From the drop-down menu, select
  :guilabel:`Instructions` to detail an instruction for the worker. To receive input from the
  workers, select the :guilabel:`Take a Picture`, :guilabel:`Register Consumed Materials`,
  :guilabel:`Print Label`, or other :ref:`quality check options <quality/quality_control_points>`.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,265 @@
+breadcrumbs. Finally, drag the last quality control line item to its intended order of instructions.
+
+.. example::
+   Drag and reorder the `Check for broken switches` instruction by clicking and dragging its "6
+   dots" icon to move it from the bottom to the second position.
```
```suggestion
   Drag and reorder the `Check for broken switches` instruction, by clicking and dragging its "6
   dots" icon to move it from the bottom to the second position.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,265 @@
+Create ECO type
+===============
+
+To access and manage ECO types, navigate to :menuselection:`PLM app --> Configuration --> ECO
+Types`. Create a new ECO type by clicking :guilabel:`New`, and fill in the :guilabel:`Name`. The
+:guilabel:`Email Alias` field is optional, but feedback submitted to the filled-in email address
+will automatically create an |ECO| in this ECO type. Modify existing ECO type names and email
+aliases by clicking on the respective ECO type in the list.
```
```suggestion
To access and manage ECO types, navigate to :menuselection:`PLM app --> Configuration --> ECO
Types`.

Create a new ECO type by clicking :guilabel:`New`, and fill in the :guilabel:`Name`. The
:guilabel:`Email Alias` field is optional, but feedback submitted to the filled-in email address
will automatically create an |ECO| in this ECO type.

Modify existing ECO type names and email aliases by clicking on the respective ECO type in the list.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,265 @@
+will automatically create an |ECO| in this ECO type. Modify existing ECO type names and email
+aliases by clicking on the respective ECO type in the list.
+
+Existing ECO types are displayed in task form on the :guilabel:`PLM Overview` page, accessible by
+navigating to :menuselection:`PLM app --> Overview`.
```
```suggestion
Existing ECO types are displayed in the task form, on the :guilabel:`PLM Overview` page, which is
accessible by navigating to :menuselection:`PLM app --> Overview`.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,265 @@
+Stage configuration
+-------------------
+
+Click an ECO type from :menuselection:`PLM app --> Overview` to open a kanban view of |ECOs| of this
+type. In the pipeline, ensure there is at least one *verification* stage, where |ECO| changes can be
+applied, and a *closing stage*, for storing completed |ECOs|. To configure a stage, hover over the
+intended stage and select the :guilabel:`⚙️ (gear)` icon. Then, click :guilabel:`Edit` to open a
+pop-up window.
```
```suggestion
Click an ECO type from :menuselection:`PLM app --> Overview` to open a kanban view of |ECOs| of this
type.

In the pipeline, ensure there is at least one *verification* stage, where |ECO| changes can be
applied, and a *closing stage*, for storing completed |ECOs|.

To configure a stage, hover over the intended stage, and select the :guilabel:`⚙️ (gear)` icon.
Then, click :guilabel:`Edit` to open a pop-up window.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,265 @@
+intended stage and select the :guilabel:`⚙️ (gear)` icon. Then, click :guilabel:`Edit` to open a
+pop-up window.
+
+Configure the verification stage in the edit stage pop-up by checking the box for :guilabel:`Allow
+to apply changes`. Optionally, add :guilabel:`Approvals` to automatically notify people to verify
+the changes before they go live. Once finished, select the :guilabel:`Save & Close` button.
```
```suggestion
Configure the verification stage in the edit stage pop-up window, by checking the box for
:guilabel:`Allow to apply changes`.

Optionally, add :guilabel:`Approvals` to automatically notify people to verify the changes before
they go live. Once finished, select the :guilabel:`Save & Close` button.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,265 @@
+
+.. example::
+
+   The stage named `Validated` is intended to store |ECOs| to be reviewed by the main approver, the
+   engineering manager, before the changes are applied to production. To reflect this, the
+   engineering manager is listed in the :guilabel:`Approvals` section. Additionally, the
+   :guilabel:`Allow to apply changes` option is checked.
```
```suggestion
   The stage titled, `Validated,` is intended to store |ECOs| that will be reviewed by the main
   approver (the engineering manager) before the changes are applied to production. To reflect this,
   the engineering manager is listed in the :guilabel:`Approvals` section. Additionally, the
   :guilabel:`Allow to apply changes` option is checked.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,265 @@
+      :align: center
+      :alt: Show "Allow to apply changes" option is checked.
+
+Next, hover over the closing stage and select the corresponding :guilabel:`⚙️ (gear)` icon to open
+the edit stage pop-up. Ensure that the :guilabel:`Folded in kanban view`, :guilabel:`Allow to apply
+changes`, and :guilabel:`Final Stage` options are checked.
```
```suggestion
Next, hover over the closing stage, and select the corresponding :guilabel:`⚙️ (gear)` icon to open
the edit stage pop-up window. Ensure that the :guilabel:`Folded in kanban view`, :guilabel:`Allow to
apply changes`, and :guilabel:`Final Stage` options are checked.
```

ksc-odoo reviewed (commented):
Alrighty @Felicious just finished with my Peer Review - fantastic work! Only had a handful of minor suggestions - but other than that, everything looks great! Nice job! 👍

Felicious commented:
Hi @ksc-odoo!! Thanks for reviewing my doc (: Do you see this message?

Felicious commented:
Hello @odoo/us-doc-review :smile: 

This PLM doc is ready for your review! Thank you in advance for your help.

Felicious commented on content/applications/inventory_and_mrp/manufacturing/plm.rst:
```diff
@@ -0,0 +1,16 @@
+:nosearch:
```
```suggestion
:nosearch:
:show-content:
:hide-page-toc:
:show-toc:
```

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+
+.. _plm/eco:
+
+Utilize *Engineering Change Orders* (*ECOs*) to track and implement changes to products, and
```
_Engineering change orders_ is not a proper noun.
Suggestion around specificity, as well (optional).

```suggestion
Utilize *engineering change orders* (*ECOs*) to track, implement, and revert change versions made to products, and
```

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+Utilize *Engineering Change Orders* (*ECOs*) to track and implement changes to products, and
+:ref:`bills of materials <manufacturing/management/bill-configuration>`.
+
+View and create :abbr:`ECOs (engineering change orders)` within each *ECO type*, a project pipeline
```
the short-hand notations at the top of the doc have the acronym definitions in title case, but here's it's not. Let's be consistent 😉

Also why not use the notation here?
```suggestion
View and create |ECOs| within each *ECO type*, a project pipeline
```

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+:ref:`bills of materials <manufacturing/management/bill-configuration>`.
+
+View and create :abbr:`ECOs (engineering change orders)` within each *ECO type*, a project pipeline
+in Gantt view with stages, ensuring progress tracking for collaborators and stakeholders.
```
Lines 15-16 have mixed subjects and don't offer a clear definition or an instruction.

StraubCreative commented:
@ksc-odoo @hojo-odoo think this could use another round of peer review so I'm kicking back to you.
There are content and proofreading issues (terms not being defined, UI not being covered, capitalization/formatting inconsistencies, etc). Can you both take another look, please?

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+   :alt: Find the tablet icon for each operation, second from the far right.
+
+Next, add an instructional step, by clicking the :guilabel:`☰ (three horizontal lines)` icon in the
+tablet view of an operation. Then, click the :guilabel:`Add a step` button.
```
This is another opportunity to expand on all the possible options that appear when this button is clicked.

Felicious replied:
This is a good idea, but many of these options are related to manufacturing orders, and only the "Add a step" button is for PLM. This is a good idea to expand on these buttons in a manufacturing doc, though!

The possible options:
![image](https://github.com/odoo/documentation/assets/23611711/ff6eba9c-fe81-4b90-aaf0-102ee80eab15)

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+   :alt: Navigate to the "Add a Step" pop-up by clicking the three horizontal lines icon in tablet
+         view.
+
+In the :guilabel:`Title` field, enter a short step description. Next, in the
```
This is where you should mention that clicking that reveals an "Add a Step" pop-up window, and so on...

Felicious replied:
Good idea for a seamless transition after the image in the middle breaks it up (:

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+         view.
+
+In the :guilabel:`Title` field, enter a short step description. Next, in the
+:guilabel:`Instructions` text field, type the instructions of the step in greater detail.
```
I believe it's just 'Instruction' field, right?

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+View ECO
+--------
+
+To review the proposed changes, go to the `BOM Changes` ECO type from :menuselection:`PLM app -->
```
I'm only seeing BOM Updates on the PLM Overview page...is that the same thing?

Felicious replied:
Yup! The cards on the PLM Overview page are like sales teams or projects. Everyone's ECO types can be different. From what I've seen, our runbots have the New Product Introduction and BOM updates eco types created by default, but sometimes they're deleted...

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+  broadly for the :guilabel:`Product`, specifically for this :guilabel:`Operation` *only*, or a
+  particular :guilabel:`Quantity` of the product
+- :guilabel:`Type`: categorizes the control point type. From the drop-down menu, select
+  :guilabel:`Instructions` to detail an instruction for the worker. To receive input from the
```
I'm not seeing an Instructions field on my run-through. Also, it may be best to just do a brief breakdown of every field (for instance, I don't see any explanation for the Team or Responsible fields, etc.)

Felicious replied:
This is tough, because this form is technically a quality control point form, and I'm not sure whether I should be exhaustive with the explanation as it's covered in the Quality documentation.  What do you think? Should I still rehash this in the context of PLM?

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+
+To access and manage ECO types, navigate to :menuselection:`PLM app --> Configuration --> ECO
+Types`.
+
```
a screenshot of a blank ECO Type form could be beneficial here.

StraubCreative replied:
To expand on that, try to fill in forms with relevant example data when possible to visually demonstrate *how* to fill in the form, and with *what* types of information 👍

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+:guilabel:`Email Alias` field is optional, but feedback submitted to the filled-in email address
+will automatically create an |ECO| in this ECO type.
+
+Modify existing ECO type names and email aliases by clicking on the respective ECO type in the list.
```
...found where?

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+Click an ECO type from :menuselection:`PLM app --> Overview` to open a kanban view of |ECOs| of this
+type.
+
+In the pipeline, ensure there is at least one *verification* stage, where |ECO| changes can be
```
might be beneficial to explain how to ensure such a thing

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+
+To configure a stage, hover over the intended stage, and select the :guilabel:`⚙️ (gear)` icon.
+Then, click :guilabel:`Edit` to open a pop-up window.
+
```
a screenshot of the Edit: New pop-up window that appears when the gear icon is clicked would be useful

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+
+Optionally, add :guilabel:`Approvals` to automatically notify people to verify the changes before
+they go live. Once finished, select the :guilabel:`Save & Close` button.
+
```
I think it would be best to expand on every field/option that appears in this pop-up (and every other pop-up and form)

ksc-odoo reviewed (commented):
Hey @Felicious I just finished another round of Peer Review on this doc (per ZST's request) and I think this doc needs a bit of sprucing up before it can move on to the next stage. The main thing that I believe you should focus on is elaboration. Explain every field in every pop-up form and detail form. Elaborate and go into step-by-step detail about everything -- even if certain fields aren't necessary. The more you explain, the less confused the reader will be. So, all that said, I think it would benefit this doc if you ran through it again -- but this time -- take your time to explain everything on every page along the way. That's my "two cents" 👍

Felicious commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+to access the new version of the |BOM|. Odoo distinguishes the non-production version of the |BOM|
+from the current version, by flagging the test |BOM| with a large :guilabel:`Archived` tag.
+
+On the new |BOM|, switch to the :guilabel:`Components` tab. Modify the components list by changing
```
Hm, yes. Let me be more specific then!

Felicious commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+To modify the operations in a |BOM|, click the :guilabel:`Revision` smart button on an active |ECO|
+to access the archived, new version of the |BOM|.
+
+In the new |BOM| version, switch to the :guilabel:`Operation Changes` tab, and make changes to the
+operations, by selecting each operation line item, and opening the corresponding :guilabel:`Open:
+Operations` pop-up window.
```
Thank you @ksc-odoo for asking all of these questions! This helps me understand what additional contextual information I can provide to explain the process clearly. Voicing any and all uncertainty you have about the flow is super welcome! :grin: I'll address your questions in my next commit and point out the lines changed (:

As for the runbot version, I've been testing everything in runbot 16.4, though I'm uncertain whether this is the right move. To my understanding, 16.3 is 16.0 with spice, and 16.4 is the beta version of 17.0. So in actuality, this doc was written for the future version in mind. I'm not sure if this is best practice, as this document is technically for version 16. Thoughts, @StraubCreative ?

Felicious replied:
Decided to remove the word "active". I initially used that adjective to describe ECOs where the "Start Revision" button has been pressed, initiating the BOM changed. But the ECOs don't spend a lot of time in the draft stage, so it didn't really make sense to make that distinction

Felicious replied:
Also, KC! The "Operations" tab is correct. I didn't notice my error, so thank you for checking for me!

Felicious commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+Apply changes
+=============
+
+After verifying the changes, move the |ECO| to a :ref:`verification stage <plm/eco/stage-config>`.
```
Do you think this rewrite is sufficient? Otherwise, I'll think again about how to define this better.

> After verifying the changes, move the |ECO| to a :ref:`verification stage <plm/eco/stage-config>`,
which are stages that require approval before the revised changes can be applied to production
|BOM|.

Felicious commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+To ensure the changes are live, navigate to the :guilabel:`Revision` smart button.
+
+The |BOM| will have the large red :guilabel:`Archived` tag removed. Additionally, navigate to the
+product form, switch to the :guilabel:`Miscellaneous` tab, and the :guilabel:`Version` field will be
```
Thank you for your thoughts! I'll improve my explanation for this part (:

Felicious commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,289 @@
+===========================
+
+|ECOs| created in *tablet view* are intended for operators to directly suggest clearer operation
+instructions. To create an |ECO|, navigate to the :guilabel:`Work Orders` tab on an ongoing
+manufacturing order. Then, click the :guilabel:`tablet icon` for the desired work order to open the
+tablet view of the operation.
```
Yes, "In Progress" is correct! Thanks for pointing out of necessity of navigational instructions here

Felicious commented:
Hi @ksc-odoo and @StraubCreative ! 
Thanks for all of your comments and suggestions. I think I was able to improve the doc a lot thanks to your discerning questions and helping me pointing out opportunities to elaborate on fields and navigational instructions. The resulting doc is much longer than before, though :sweat_smile: , so I split it into two docs, engineering change orders, and ECO type.

I was also trying to push this before going on vacation, so there might be errors that I missed :skull: 

Because this is a long doc, I wanted to summarize the changes with the specific lines so it'd be easier for you to review and see whether I addressed your comments! Again, I really appreciate your reviews :smile: 

### Changelog
- Explained all of the fields of an ECO 42-70.
- Add screenshot of archived BOM (line 88 - 95)
- Add explanation of each Operation changes tab column (lines 154 - 212)
- Improved explanation of a verification stage (219-221)
- Improved explanation of what happens when Apply Changes button is pressed (223-225)
- Improved explanation of how to navigate to the product form > BOM >  to check the Version field in the Misc tab (lines 227 - 239)
- Improved navigational instructions of getting to the tablet view (lines 256 - 266)
- Add clarification about how ECOs are generated from the “Add a Step” window (298 - 306)
- Add clarification for viewing ECOs created from tablet view (311-320)
- Move ECO type to new doc and clarify verification and closing stage definitions

StraubCreative commented:
@ksc-odoo do you approve the changes?
@hojo-odoo can you review the content please?

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm.rst:
```diff
@@ -0,0 +1,20 @@
+Product lifecycle management
+============================
+
+Odoo *Product Lifecycle Management* (*PLM*) offers a systematic approach for testing, collaboration,
+and iteration of products across concept development, design, manufacturing, marketing, and
+post-launch support stages.
```
```suggestion
Odoo *Product Lifecycle Management* (*PLM*) offers a systematic approach for the testing,
collaborating, and iterating of products across concept development, design, manufacturing,
marketing, and post-launch support stages.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/eco_type.rst:
```diff
@@ -0,0 +1,108 @@
+ECO type
+========
+
+Organize engineering change orders (ECOs) into a *ECO type* project and create custom stages to
+track the progress of changes within a structured Gantt view project pipeline. This categorization
+ensures that collaborators and stakeholders only view and assist with relevant |BOM| improvements.
```
```suggestion
Organize engineering change orders (ECOs) into an *ECO type* project, and create custom stages to
track the progress of changes within a structured Gantt view project pipeline. This categorization
ensures collaborators and stakeholders **only** view and assist with relevant |BOM| improvements.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/eco_type.rst:
```diff
@@ -0,0 +1,108 @@
+Create a new ECO type by clicking :guilabel:`New`. On the new :guilabel:`ECO Types` form, fill in
+the following information:
+
+- :guilabel:`Name`: the name of the |ECO| and will organize all of the |ECOs| of this *type* in a
+  project.
```
```suggestion
- :guilabel:`Name`: the name of the |ECO|, which will organize all of the |ECOs| of this *type* in a
  project.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/eco_type.rst:
```diff
@@ -0,0 +1,108 @@
+
+- :guilabel:`Name`: the name of the |ECO| and will organize all of the |ECOs| of this *type* in a
+  project.
+- :guilabel:`Email Alias`: an optional field. If filled, emails submitted to this email address will
+  automatically generate |ECOs| in the left-most stage of this |ECO| type.
```
```suggestion
- :guilabel:`Email Alias`: if this optional field is filled, emails submitted to this email address
  automatically generate |ECOs| in the left-most stage of this |ECO| type.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/eco_type.rst:
```diff
@@ -0,0 +1,108 @@
+Stage configuration
+===================
+
+*Stages* in each |ECO| type help to organize and track the progress of |ECOs|. Additionally,
+approvers can be configured on each stage to enforce reviews on changes and prevent errors on the
+production |BOM|.
```
```suggestion
*Stages* in each |ECO| type help organize and track the progress of |ECOs|. Additionally, approvers
can be configured on each stage to enforce reviews on changes, and prevent errors on the production
|BOM|.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/eco_type.rst:
```diff
@@ -0,0 +1,108 @@
+approvers can be configured on each stage to enforce reviews on changes and prevent errors on the
+production |BOM|.
+
+For best practice, at there is at least one *verification* stage, which is a stage with a required
+approver, and one *closing* stage, which stores |ECOs| that have been either canceled or approved
+for use as the next production |BOM|.
```
```suggestion
For best practice, there should be at least one *verification* stage, which is a stage with a
required approver, and one *closing* stage, which stores |ECOs| that have been either canceled or
approved for use as the next production |BOM|.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/eco_type.rst:
```diff
@@ -0,0 +1,108 @@
+Configure the verification stage in the edit stage pop-up window, by checking the box for
+:guilabel:`Allow to apply changes`.
+
+Add an approver in the :guilabel:`Approvers` section by clicking :guilabel:`Add a line` and
+specifying the :guilabel:`Role` of the reviewer, their :guilabel:`User`, and :guilabel:`Approval
+Type`. Ensure at least one approver is configured with the :guilabel:`Approval Type`, :guilabel:`Is
+required to approve`.
```
```suggestion
Then, add an approver in the :guilabel:`Approvers` section, by clicking :guilabel:`Add a line`, and
specifying the :guilabel:`Role` of the reviewer, their :guilabel:`User`, and :guilabel:`Approval
Type`.

Make sure at least one approver is configured with the :guilabel:`Approval Type`: :guilabel:`Is
required to approve`.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/eco_type.rst:
```diff
@@ -0,0 +1,108 @@
+
+.. example::
+   In the |ECO| type `New Product Introduction`, the verification stage `Validated` is configured by
+   clicking :guilabel:`⚙️ (gear)` icon and selecting :guilabel:`Edit`. Doing so opens the
+   :guilabel:`Edit: Validated` pop-up window.
```
```suggestion
   clicking the :guilabel:`⚙️ (gear)` icon, and selecting :guilabel:`Edit`. Doing so opens the
   :guilabel:`Edit Validated` pop-up window.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/eco_type.rst:
```diff
@@ -0,0 +1,108 @@
+   :guilabel:`Edit: Validated` pop-up window.
+
+   By adding the `Engineering manager` as an approver, only |ECOs| approved by this user can proceed
+   to the next stage and have the changes applied on the production |BOM|.
```
```suggestion
   to the next stage, and have the changes applied on the production |BOM|.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/eco_type.rst:
```diff
@@ -0,0 +1,108 @@
+   By adding the `Engineering manager` as an approver, only |ECOs| approved by this user can proceed
+   to the next stage and have the changes applied on the production |BOM|.
+
+   Additionally, check the :guilabel:`Allow to apply changes` option ensure proper behavior.
```
```suggestion
   Additionally, check the :guilabel:`Allow to apply changes` option to ensure proper behavior.
```

ksc-odoo reviewed (commented):
@StraubCreative Just gave this another review -- looks like there's still a handful of portions that need @Felicious 's attention. But, once those areas are given a second look, I'm sure it'll be ready for the next stage! 👍

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,369 @@
+  products in a specific company, or leave blank if the change applies to all companies.
+- :guilabel:`Responsible`: assignee in charge of this |ECO|. (Optional)
+- :guilabel:`Effective`: specifies when the |ECO| becomes live. Choosing :guilabel:`As soon as
+  possible` means that the |ECO| applies to the production |BOM| as soon as an authorized user
```
```suggestion
  possible` means the |ECO| applies to the production |BOM| as soon as an authorized user
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,369 @@
+- :guilabel:`Responsible`: assignee in charge of this |ECO|. (Optional)
+- :guilabel:`Effective`: specifies when the |ECO| becomes live. Choosing :guilabel:`As soon as
+  possible` means that the |ECO| applies to the production |BOM| as soon as an authorized user
+  :ref:`applies the changes <plm/eco/apply-changes>`. On the other hand, choosing :guilabel:`At
+  Date` and setting a specific date leaves a date that makes it easier to track the version history
+  of the |BOM| and tracking the specific date |BOMs| are used for production.
```
```suggestion
  :ref:`applies the changes <plm/eco/apply-changes>`.
  
  On the other hand, choosing :guilabel:`At Date`, and setting a specific date, leaves a date that
  makes it easier to track the version history of the |BOM|, and the specific date |BOMs|, used for
  production.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,369 @@
+By pressing :guilabel:`Start Revision`, three actions occur:
+
+#. The :guilabel:`Documents` smart button appears, storing relevant files of the |BOM|.
+#. A copy of the production |BOM| is stored in the newly appeared :guilabel:`Revision` smart button
```
```suggestion
#. A copy of the production |BOM| is stored in the newly-appeared :guilabel:`Revision` smart button
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,369 @@
+#. The stages of the |ECO| :guilabel:`Type` are displayed in the top-right corner of the |ECO|.
+
+.. note::
+   The :guilabel:`Revision` smart button is available only when the :guilabel:`Bill of Materials`
+   radio button is selected in the :guilabel:`Apply on` field and the :guilabel:`Start Revision`
+   button has been pressed.
```
```suggestion
   The :guilabel:`Revision` smart button is available **only** when the :guilabel:`Bill of
   Materials` radio button is selected in the :guilabel:`Apply on` field, and the :guilabel:`Start
   Revision` button has been pressed.
```

hojo-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/eco_type.rst:
```diff
@@ -0,0 +1,108 @@
+Edit ECO type
+=============
+
+Modify existing |ECO| type names and email aliases by clicking by navigating to the
+:menuselection:`PLM app --> Configuration --> ECO Types` page. There, click on the desired on the
```
```suggestion
Modify existing |ECO| type names and email aliases by navigating to
:menuselection:`PLM app --> Configuration --> ECO Types` . There, click on the desired on the
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,369 @@
+      :align: center
+      :alt: Show the archived Bill of Materials.
+
+On the new |BOM|, in the :guilabel:`Components` tab. Modify the components list by changing the
+:guilabel:`Quantity` of existing components, add new components using the :guilabel:`Add a line`
+button, and remove components with the :guilabel:`🗑️ (trash)` icon.
```
```suggestion
On the new |BOM|, in the :guilabel:`Components` tab, proceed to modify the components list, by
changing the :guilabel:`Quantity` of existing components, adding new components using the
:guilabel:`Add a line` button, and removing components with the :guilabel:`🗑️ (trash)` icon.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,369 @@
+
+Blue text indicates components newly added to the revised |BOM| that do not yet exist in the
+production |BOM|. Black text represents updates shared by both |BOMs|, while red text indicates
+components present in the production |BOM| but removed in the revised |BOM|.
```
```suggestion
components present in the production |BOM|, but removed in the revised |BOM|.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,369 @@
+
+Blue text indicates operations newly added to the revised |BOM| that do not yet exist in the
+production |BOM|. Black text represents updates shared by both |BOMs|, while red text indicates
+operations present in the production |BOM| but removed in the revised |BOM|.
```
```suggestion
operations present in the production |BOM|, but removed in the revised |BOM|.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,369 @@
+Modifications to the |BOM| in an |ECO| will **not** affect the |BOM| used in production. That is,
+until the :ref:`changes are applied <plm/eco/apply-changes>`.
+
+In the :guilabel:`Operation Changes` tab, each row of details the columns in the table reflect the
+following information:
```
```suggestion
In the :guilabel:`Operation Changes` tab, each row of details, beneath the columns in the table,
reflect the following information:
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,369 @@
+following information:
+
+- :guilabel:`Operation`: Name of the operation that was modified.
+- :guilabel:`Step` column specifies the quality control point, visible when the operation includes
+  detailed instructions.
```
```suggestion
- :guilabel:`Step`: specifies the quality control point, visible when the operation includes
  detailed instructions.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,369 @@
+
+.. note::
+   To check for instructions, click the operation line item in the :guilabel:`Operations` tab of a
+   |BOM|. Then, in the :guilabel:`Open: Operations` pop-up, look for the :guilabel:`Instructions`
```
```suggestion
   |BOM|. Then, in the :guilabel:`Open: Operations` pop-up window, look for the :guilabel:`Instructions`
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,369 @@
+      :alt: Show *Instructions* smart button to check whether an operation has additional
+            instructions.
+
+- :guilabel:`Step Type` details the type of quality control for further instructions in the
+  operation.
+
+- :guilabel:`Type` field corresponds with the colored text to specify how the revised |BOM| differs
+  from the production |BOM|. Operation change types can be :guilabel:`Add`, :guilabel:`Remove`, or
+  :guilabel:`Update`.
+- :guilabel:`Work Center`: specifies the work center the operation is performed at.
+- :guilabel:`Manual Duration Change`: refers to the change in the :guilabel:`Default Duration` field
+  in the :guilabel:`Open: Operations` pop-up, which specifies the expected time for completing the
+  operation.
```
```suggestion
- :guilabel:`Step Type` details the type of quality control for further instructions in the
  operation.
- :guilabel:`Type` corresponds with the colored text to specify how the revised |BOM| differs
  from the production |BOM|. Operation change types can be :guilabel:`Add`, :guilabel:`Remove`, or
  :guilabel:`Update`.
- :guilabel:`Work Center` specifies the work center at which the operation is performed.
- :guilabel:`Manual Duration Change` refers to the change in the :guilabel:`Default Duration` field
  in the :guilabel:`Open: Operations` pop-up window, which specifies the expected time for completing the
  operation.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,369 @@
+=============
+
+After verifying the changes, move the |ECO| to a :ref:`verification stage <plm/eco/stage-config>`,
+which are stages that require approval before the revised changes can be applied to production
+|BOM|.
```
```suggestion
which are stages that require approval before the revised changes can be applied to the
production |BOM|.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,369 @@
+
+Once the approvers accept the changes, the :guilabel:`Apply Changes` button becomes available. Click
+this button, and the |ECO| is automatically moved to a closing stage. The changes are applied, which
+archives the original, production |BOM|, and the revised |BOM| becomes the new production |BOM|.
```
```suggestion
archives the original production |BOM|, and the revised |BOM| becomes the new production |BOM|.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,369 @@
+Verify changes
+--------------
+
+To ensure the changes are live, from the |ECO| that where the :guilabel:`Apply Changes` button was
+just pressed, return to the revised |BOM| by clicking the :guilabel:`Revision` smart button.
```
```suggestion
To ensure the changes are live, from the |ECO| where the :guilabel:`Apply Changes` button was
just pressed, return to the revised |BOM| by clicking the :guilabel:`Revision` smart button.
```

hojo-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/eco_type.rst:
```diff
@@ -0,0 +1,108 @@
+===================
+
+*Stages* in each |ECO| type help to organize and track the progress of |ECOs|. Additionally,
+approvers can be configured on each stage to enforce reviews on changes and prevent errors on the
```
Are they really enforcing reviews or are they approving suggested changes? I think the phrasing here could be reworked.

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,369 @@
+On the :ref:`quality control point <quality/quality_control_points>` form, ensure the following form
+fields are accurately filled out to give detailed instructions for operators:
+
+- :guilabel:`Title`: rename to give a concise description of the new instruction
```
```suggestion
- :guilabel:`Title`: rename to give a concise description of the new instruction.
```

hojo-odoo commented on content/applications/inventory_and_mrp/manufacturing/plm/eco_type.rst:
```diff
@@ -0,0 +1,108 @@
+-------------
+
+To configure a closing stage, hover over the closing stage, and select the corresponding
+:guilabel:`⚙️ (gear)` icon to open the edit stage pop-up window. Ensure that the :guilabel:`Folded
```
```suggestion
:guilabel:`⚙️ (gear)` icon to open the :guilabel:`Edit: [stage]` pop-up window. Ensure that the :guilabel:`Folded
```

hojo-odoo reviewed (commented):
Hey @Felicious,

I left a bunch of comments on the first doc in this PR, but before I look at the rest, I think you should do an in-depth review of all of the content from beginning to end. I'm finding a lot of grammar and formatting errors, as well as content that seems incorrect. Please re-request my review when you're done and I'll take another look.

Thanks,
hojo

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/plm/eco_type.rst:
```diff
@@ -0,0 +1,108 @@
+ECO type
+========
+
+Organize engineering change orders (ECOs) into a *ECO type* project and create custom stages to
```
We typically use *italics* for initialisms, not capital letters.

The italics are more appropriate, I'd argue, because:
- stylistically, they are is as effective as capital letters in visually communicating a difference in the text, meaning, they clue in the reader to pay attention;
- orthographically, they are correct since they are actually emphasizing something, like italics is supposed to do. Whereas, the capitalization is a bit more of a subjective use case in stylizing the initialisms (they're not proper nouns, after all), no?.
- consistently, this is what we typically do on doc for acronym,  so whether or not it's the best or most correct way, it is at least...consistent.

Whether or not we agree on this, I appreciate you for giving me the excuse to flex _orthography_— that one had some dust on it! 😅 These, by the way, are definitely the nuanced ideas/discussions I like to see, so cheers to that! 🍻

Felicious commented:
Hi @hojo-odoo and @ksc-odoo 

Sorry and thank you for looking at this monster of a doc. KSC left a ton of really good feedback, and since I was leaving for vacation, I panicked and shoved all the extra info that *I thought* addressed his comments into the doc. The result: a ~280 line doc became two docs with a combined ~500 line count :skull: 

Because there were a TON of changes and it's been more than a month since your last review, I made a [changelog](https://github.com/odoo/documentation/pull/5238#issuecomment-1732225892) after I pushed up changes. Not sure if it helped with much?

Since we have to squash all of our commits, the version of the doc you two saw gets lost, so I'm sure it's quite confusing to read the doc after all the changes have been made. I tried to dig in runbot to recover the version each of you reviewed, improved, and approved, but it's been more than 6 weeks so the runbots have been killed. 

- (e.g. This is the [runbot for the version](https://runbot.odoo.com/runbot/batch/1145020) you saw, @hojo-odoo 
- And this is [what you approved](https://runbot.odoo.com/runbot/batch/1146445), @ksc-odoo. but these links are both dead now :sob: 

But fortunately, KSC, I was able to dig up the version of the doc you saw during your second review, before I split the docs into two pages. [Linked here](https://runbot174.odoo.com/runbot/static/build/51147137-16-0/logs/build/html/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.html) 

It's not ideal to use this to compare versions and see whether my changes addressed your comments, but it's the best we can do since I already squashed the previous commits :cry: Maybe going forward, if I make a ton of changes, I'll squash commits into two instead of one: so the review can compare with the doc they initially read/reviewed to the new version the writer pushed up to address the comments the reviewer left? :smile: I can bring this up in a 1:1 with @StraubCreative!

Anyways I'll work on this PR for the rest of today. Appreciate you both for helping me with this doc while I was gone! :grinning:

Felicious commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,265 @@
+
+Engineering change orders can be created:
+
+#. :ref:`directly in the ECO type <plm/eco/create-eco>`
+#. by an operator in the :ref:`tablet view <plm/eco/tablet-view>` of an operation
+#. automatically from feedback submitted to the :ref:`ECO type's email alias <plm/eco/eco-type>`.
```
```suggestion
#. :ref:`directly in the ECO type <plm/eco/create-eco>`.
#. by an operator in the :ref:`tablet view <plm/eco/tablet-view>` of an operation.
#. automatically from feedback submitted to the :ref:`ECO type's email alias <plm/eco/eco-type>`.
```

Felicious commented on content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst:
```diff
@@ -0,0 +1,369 @@
+
+Engineering change orders can be created:
+
+#. :ref:`directly in the ECO type <plm/eco/create-eco>`
+#. by an operator in the :ref:`tablet view <plm/eco/tablet-view>` of an operation
+#. automatically from feedback submitted to the :ref:`ECO type's email alias <plm/eco/eco-type>`.
```
```suggestion
#. :ref:`directly in the ECO type <plm/eco/create-eco>`.
#. by an operator in the :ref:`tablet view <plm/eco/tablet-view>` of an operation.
#. automatically from feedback submitted to the :ref:`ECO type's email alias <plm/eco/eco-type>`.
```

Felicious commented:
Hi @ksc-odoo and @hojo-odoo! 

I implemented all of your suggestions and the PR is ready for your review! I still have some scattered errors that I'm currently working on resolving, but the content is ready for review (:

I also didn't squash two of the commits-- so you could see the version of the doc BEFORE your suggestions (the "[ADD]..." commit) and the version AFTER your suggestions (the "KSC and John's suggestions" commit). You can view the comparison of the two versions in the "Split" view of the second commit. [Here's a link to get there](https://github.com/odoo/documentation/pull/5238/commits/91227adb9fba81f1192625f32d33302e2658fee7)

Change to split view by clicking the gear icon and selecting split
![image](https://github.com/odoo/documentation/assets/23611711/c9edd606-2bad-4e8a-b37c-f99d4e806e08)

Thanks for taking another look!

samueljlieber commented:
Hi @AntoineVDV and @odoo/user-docs-structure-review I believe we will need to update the code-owners for this new addition (`plm/*`) to `inventory_and_mrp/`. 

Thank you!

StraubCreative commented:
> Hi @AntoineVDV and @odoo/user-docs-structure-review I believe we will need to update the code-owners for this new addition (`plm/*`) to `inventory_and_mrp/`.

I just created the PLM doc review team [here](https://github.com/orgs/odoo/teams/plm-doc-review/members)

StraubCreative commented:
> I also didn't squash two of the commits-- so you could see the version of the doc BEFORE your suggestions (the "[ADD]..." commit) and the version AFTER your suggestions (the "KSC and John's suggestions" commit). You can view the comparison of the two versions in the "Split" view of the second commit. [Here's a link to get there](https://github.com/odoo/documentation/pull/5238/commits/91227adb9fba81f1192625f32d33302e2658fee7)

Hi, the link here ☝️ doesn't work actually.
Here's a more general way to see the diff between the two commits: https://github.com/odoo/documentation/compare/7e9c703..b19c356

However this seems unnecessary— let's have reviewers just look at whatever is the latest. 
Multiple commits are more ideal for code / feature exploration, not so much markup like this.

WIth that in mind, @Felicious can you squash everything down so @ksc-odoo and @hojo-odoo have 1 clean commit to look at, please?

jcs-odoo commented:
> > Hi @AntoineVDV and @odoo/user-docs-structure-review I believe we will need to update the code-owners for this new addition (`plm/*`) to `inventory_and_mrp/`.
> 
> I just created the PLM doc review team [here](https://github.com/orgs/odoo/teams/plm-doc-review/members)

I don't have the rights to do it, but I'll find someone who can :)

Felicious commented:
Hi @StraubCreative! Just resolved the merge conflicts, and the other issues have been resolved too (Thanks, Jonathan). This PR is ready for your review!

StraubCreative commented on content/applications/inventory_and_mrp.rst:
```diff
@@ -13,3 +13,4 @@ Inventory & MRP
     inventory_and_mrp/barcode
     inventory_and_mrp/quality
     inventory_and_mrp/maintenance
+    inventory_and_mrp/plm
```
Small thing I noticed:
The toctree links in this doc aren't exactly lined up with the indentation of the tag 😄

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst:
```diff
@@ -1,15 +1,15 @@
-A *Bill of Materials* (or *BoM* for short) is a document that defines the quantity of each
-component required to make or deliver a finished product. It can also include various operations
-and the individual step guidelines needed to complete a production process.
+A *Bill of Materials* (or *BoM* for short) is a document that defines the quantity of each component
```
bill of materials is not a proper noun

```suggestion
A *bill of materials* (or *BoM* for short) is a document that defines the quantity of each component
```

StraubCreative commented on content/applications/inventory_and_mrp/plm/manage_changes.rst:
```diff
@@ -0,0 +1,9 @@
+.. toctree::
+   :titlesonly:
+
+   manage_changes/eco_type
+   manage_changes/engineering_change_orders
```
would switch the order of these since ECO doc introducing the concept would ideally come first.
fwiw doc content is **not** organized alphabetically in English.

StraubCreative commented on content/applications/inventory_and_mrp/plm/manage_changes/engineering_change_orders.rst:
```diff
@@ -0,0 +1,378 @@
+   Step` pop-up window. Doing so creates an instructional quality control point that will be
+   reviewed in the following section.
+
+   .. image:: engineering_change_orders/add-a-step.png
```
is this image compressed?

Felicious replied:
no, updating! thanks for catching that

StraubCreative commented on content/applications/inventory_and_mrp/plm/manage_changes/engineering_change_orders.rst:
```diff
@@ -0,0 +1,378 @@
+   Materials` radio button is selected in the :guilabel:`Apply on` field, and the :guilabel:`Start
+   Revision` button has been pressed.
+
+.. image:: engineering_change_orders/eco-form.png
```
same here— is this compressed?

Felicious replied:
thanks for catching that!

StraubCreative commented on content/applications/inventory_and_mrp/plm/manage_changes/engineering_change_orders.rst:
```diff
@@ -0,0 +1,378 @@
+   Drag and reorder the `Check for broken switches` instruction, by clicking and dragging its "6
+   dots" icon to move it from the bottom to the second position.
+
+   .. image:: engineering_change_orders/reorder.png
```
this one doesn't appear to be compressed given how small the dimensions are

StraubCreative reviewed (approved):
@Felicious approving with some small comments.
Merge when ready, thanks!

@robodoo delegate=Felicious

Felicious commented:
@robodoo r+
